### PR TITLE
fix(core): do not load GA script if not configured

### PR DESCRIPTION
**Pull request by anotherchrisberry**
Not that this is the biggest contributor to slow load times for Deck, but...might as well not load the Google Analytics script if it's not going to be used?

### CODE REVIEW

File: app/index.deck [hunk 1]
```diff
@@ -175,11 +175,14 @@
   </script>
 
   <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
+    window.addEventListener('load', () => {
+      if (window.spinnakerSettings && window.spinnakerSettings.analytics.ga) {
```
Comment (erikmunson) [inline app/index.deck]: do we support any other fields in `analytics` that are used without `ga`?

Comment (anotherchrisberry) [reply app/index.deck]: yeah, good catch

**Comment by ttomsu** [comment]
LGTM

**Comment by ttomsu** [comment]
@spinnakerbot <https://github.com/spinnakerbot> cherry-pick 1.11

**Comment by duftler** [comment]
Manual cherry-pick: https://github.com/spinnaker/deck/pull/6196

**Pull request merged**